src/subsystems/python/builders/simple-python/default.nix

```diff
@@ -40,40 +40,40 @@
     package = produceDerivation defaultPackageName (buildFunc {
       name = defaultPackageName;
       src = getSource defaultPackageName defaultPackageVersion;
-      format = "setuptools";
+      format = "other";
```

Comment:
Yes, with pleasure! Maybe it would be useful to do a call or at least open up a pad to collect requirements in a table or so and try to see where those overlap? @chaoflow 

I quickly experimented with just building a wheel for poetry projects a well and install that one in an extra step. That currently fails if the project being packaged has a dependency which is also (transitively) depended on by buildInputs, because that it tries to uninstall the former one first. I think going that route should allow us to ship most of the code? But not yet sure how to solve that

Reply:
Sure, let's have a call. How about we use the dream2nix weekly tomorrow (Tuesday) at 14:00 for this?

Reply:
@phaer @DavHau I'm in as well.